backend/src/main/java/com/huellapositiva/application/controller/ProposalApiController.java

```diff
@@ -544,5 +553,52 @@ public void changeStatusVolunteerInProposal(@RequestBody List<ChangeStatusVolunt
     public void changeReviewPendingProposalToPublished(@RequestBody ChangeStatusProposalRequestDto dto) {
         changeReviewPendingProposalToPublishedAction.execute(dto.getIdProposal());
     }
-}
 
+    @Operation(
+            summary = "Updates the proposal image",
+            description = "The contact person can update the image of the proposal.",
+            tags = {"proposals, contact person"},
+            parameters = {
+                    @Parameter(name = "X-XSRF-TOKEN", in = ParameterIn.HEADER, required = true, example = "ff79038b-3fec-41f0-bab8-6e0d11db986e", description = "For taking this value, open your inspector code on your browser, and take the value of the cookie with the name 'XSRF-TOKEN'. Example: a6f5086d-af6b-464f-988b-7a604e46062b"),
```

Comment:
CSRF token is sent as a cookie and a query parameter. Check the Postman collection in the project from the usage

`ParameterIn.QUERY`

Reply:
Done